File: src/main/java/fr/benjamindanlos/laptimes/LaptimesApplication.java
package fr.benjamindanlos.laptimes;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.scheduling.annotation.EnableAsync;

@Configuration
@EnableAutoConfiguration(exclude = SecurityAutoConfiguration.class)
@ComponentScan
@EnableAsync
public class LaptimesApplication {
	public static void main(String[] args) {
		SpringApplication.run(LaptimesApplication.class, args);
	}
}

File: src/main/kotlin/fr/benjamindanlos/laptimes/LaptimesApplication.kt
package fr.benjamindanlos.laptimes

import org.springframework.boot.autoconfigure.EnableAutoConfiguration
import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration
import org.springframework.boot.runApplication
import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.Configuration

@Configuration
@EnableAutoConfiguration(exclude = [SecurityAutoConfiguration::class])
@ComponentScan
class LaptimesApplication

fun main(args: Array<String>) {
	//System.out.println(InetAddress.getLocalHost())
	//val listener = Listener(9999)
	//listener.run()
	runApplication<LaptimesApplication>(*args)
}
